GOVERNANCE.md

```diff
@@ -28,76 +27,72 @@ Flux strives to operate independently of specific partisan interests, and for de
 
 ### Code of Conduct
 
-The Flux community adheres to the CNCF Code of Conduct <https://github.com/cncf/foundation/blob/master/code-of-conduct.md>.
+[Code of Conduct](CODE_OF_CONDUCT.md) violations by community members will be discussed and resolved on the [private Maintainer mailing list](mailto:cncf-flux-maintainers@lists.cncf.io). If the reported CoC violator is a Maintainer, the Maintainers will instead designate two Maintainers to work with CNCF staff in resolving the report.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a _Flux_ [Oversight Committee] member.
+### Meetings
 
-If no conclusion can be reached in meditation, such issues can be escalated to the CNCF mediator, Mishi Choudhary <mishi@linux.com>, in which case CNCF may choose to intervene.
+Time zones permitting, Maintainers are expected to participate in the [public
+developer meeting](README.md#meetings).
 
-## Roles in the Flux Community
+Maintainers will also have closed meetings in order to discuss security reports
+or Code of Conduct violations.  Such meetings should be scheduled by any
+Maintainer on receipt of a security issue or CoC report.  All current Maintainers
+must be invited to such closed meetings, except for any Maintainer who is
+accused of a CoC violation.
 
-See [community-roles.md]
+## Roles and Process in the Flux Community
+
+See
+
+- [community-roles.md] for the definition of roles in our community and
+- [PROCESS.md] to understand how to interact with Flux Community processes
 
 ## Decision Making
 
+While we spell out all the possibilities for decision making and possible eventualities below, the Flux project almost always looked for and found consensus. Using lazy consensus never resulted in later disputes. Our communication channels (public meetings, Slack, RFCs, etc) provide us with the means to have agreement before we set up a vote.
+
+If we fail to make a decision or the picture was unclear, we found that we needed more information. In these cases, we reach out to users, adjacent projects or counterpart teams in the CNCF.
+
 ### Deciders
 
 - Repository [Maintainers][Maintainer]: Decisions that affect only one git repository.
-- [Oversight Committee]: Decisions that are outside the scope of a single git repository.
+- [flux2 maintainers]: Decisions that are outside the scope of a single git repository.
```

Comment:
I would change the name to **core maintainers** since the members of this group will have admin privileges of the whole organization and all Flux related projects.

Reply:
Are we going to create a team called https://github.com/orgs/fluxcd/teams/flux2-maintainers ... or rename it?

Reply:
I would rename the current one, after this PR is merged.